src/sentry/search/events/datasets/discover.py

```diff
@@ -222,6 +223,17 @@ def function_converter(self) -> Mapping[str, SnQLFunction]:
                     ),
                     default_result_type="integer",
                 ),
+                SnQLFunction(
+                    "count_web_vitals",
+                    required_args=[
```

Comment:
Nit: To be consistent with functions like `percentile`, can we have the column as the first argument and the quality as the second?